_data/changelogs/side-navigation.yml

```diff
@@ -19,4 +19,49 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+  # 3.1.0 release
+- date: 2022-08-05
+  summary: |-
+    Non-form buttons inside of usa-sidenav require `type="button"` to prevent default `submit` behaviors.
```

Comment:
The backticks don't render to code elements. This renders as:

**Before**
![image](https://user-images.githubusercontent.com/3385219/199806762-5a65102e-f1bb-4f06-bd02-3de5161586f1.png)

If we add the `markdownify` filter to [`changelog-table.html`](https://github.com/uswds/uswds-site/blob/d38522c1f8e9a6029d75c8ffbf5c3152c424a497/_includes/changelog-table.html#L88) it'll render, but in separate paragraphs.

**After**
![image](https://user-images.githubusercontent.com/3385219/199807041-6efefb4a-ca3b-473f-8d58-f78cbed90b25.png)

Reply:
This is a smart addition - it would help make future data entry easier. Thanks!

I went ahead and committed a change to this branch (since it is the only one that needs it atm) to add markdownify to the template. Setting markdownify to remove the `<p>` tags should resolve the issue with separate paragraphs, but let me know if you see any possible pitfalls with this approach. 

[78f7276](https://github.com/uswds/uswds-site/pull/1871/commits/78f7276a5cae51e430260432a288aea212baa382)

Reply:
Glad you like the idea! Your fix seems like a nice solution for these summaries.